app/routes/page.tsx

```
@@ -0,0 +1,78 @@
+            {data && (
+              <>
+                <RemixHead>
+                  <title>{`${meta.title} - DevRelKaigi 2025`}</title>
+                </RemixHead>
+                <Breadcrumb items={[
+                  { label: t('Home'), href: `/${locale}` },
+                  { label: meta.title! }
+                ]} />
+                <div
+                  className="article-body"
+                  dangerouslySetInnerHTML={{ __html: md.render(body as string) }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)